spec/system/posts_index_page_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+RSpec.describe 'Post/Index', type: :system do
+  let(:user) do
+    User.create(name: 'Harry', photo: 'https://photos.com/harrystyles', bio: 'harry Styles here!')
+  end
+
+  let(:post) do
+    Post.create(author: user, text: 'text', title: 'First!')
+  end
+
+  let(:post2) do
+    Post.create(author: user, text: 'text', title: 'Second!')
+  end
+
+  let(:post3) do
+    Post.create(author: user, text: 'text', title: 'Third!')
+  end
+
+  let(:like) do
+    Like.create(user:, post:)
+  end
+
+  let(:comment) do
+    Comment.create(user:, post:, text: 'text')
+  end
+
+  before :each do
+    user.save
+    post.save
+    post2.save
+    post3.save
+    like.save
+    comment.save
+  end
+
+  describe 'Index page' do
+    it 'I can see the user\'s profile pic.' do
+      visit user_posts_path(user)
+      expect(page).to have_css("img[src*='#{user.photo}']")
+    end
+
+    it 'I can see the user\'s profile name.' do
+      visit user_posts_path(user)
+      expect(page).to have_content(user.name)
+    end
+
+    it 'I can see the user\'s post count.' do
+      visit user_posts_path(user)
+      expect(page).to have_content(user.post_counter)
+    end
+
+    it 'I can see a post title.' do
+      visit user_posts_path(user)
+      expect(page).to have_content(post.title)
+    end
+
+    it 'I can see some of post body.' do
+      visit user_posts_path(user)
+      expect(page).to have_content(post.text)
+      expect(page).to have_content(post2.text)
+      expect(page).to have_content(post3.text)
+    end
+
+    it 'I can see a post comment.' do
+      visit user_posts_path(user)
+      expect(page).to have_content(comment.text)
+    end
```

Comment:
@Team great work adding tests for the posts' index page, however, this test is supposed to check if the `recent_comments` method is working correctly, you can add more comments for this test case, then check if the method displays only the 5 recent comments.